### DISCUSSION TITLE
getaddrinfo ENOTFOUND Error

### DISCUSSION THREAD

**Discussion by arpit314ad** [Budibase/budibase#8462] [Help]
Hi...

I am trying to connect MS SQL Server DB but it is fails. It gives an error which is mentioned above the subject line.

Host : Windows10
SQL: Microsoft SQL Server 2016 (SP2) (KB4052908) 
SSMS: 18.123.1 version

Please look into the same.

![SQL Connect Error](https://user-images.githubusercontent.com/86228170/199223948-862dba1e-20b5-4b15-9301-251881ceb191.JPG)

**Comment by andz-bb** [comment]
Hey @arpit314ad, can you check that TCP/IP is enabled for your SQL Server and that the firewall isn't blocking port 1433?

**Reply by arpit314ad** [reply]
Hi...
TCP/IP is enable. The firewall settings are also off.

![Firewal](https://user-images.githubusercontent.com/86228170/199405315-933df708-7cb7-4c14-a409-1f8bd03210ee.JPG)

![TCP-IP](https://user-images.githubusercontent.com/86228170/199405272-8261bd2b-55a9-46b7-8533-3bd9cd3c1de0.JPG)

**Reply by andz-bb** [reply]
@arpit314ad could you try using the IP address for the server instead of STTLSERVICE01?

**Reply by arpit314ad** [reply]
Hi... Andrew,

I have tried with IP but it gives an error.

Thanks for your reply. 

![IP Error](https://user-images.githubusercontent.com/86228170/199656293-add23660-a201-4209-83d9-5b1d3bb65fcf.JPG)
